src/wizard/wizard.component.js

```diff
@@ -3,27 +3,27 @@ angular.module('patternfly.wizard').component('pfWizard', {
   bindings: {
     title: '@',
     wizardTitle: '@',
-    hideIndicators: '=?',
+    hideIndicators: '<?',
     activeStepTitleOnly: '<?',
     hideSidebar: '@',
     hideHeader: '@',
     hideBackButton: '@',
     sidebarClass: '@',
     stepClass: '@',
-    contentHeight: '=?',
+    contentHeight: '<?',
     currentStep: '<?',
-    cancelTitle: '=?',
-    backTitle: '=?',
-    nextTitle: '=?',
-    backCallback: '=?',
-    nextCallback: '=?',
+    cancelTitle: '<?',
+    backTitle: '<?',
+    nextTitle: '<?',
+    backCallback: '<?',
+    nextCallback: '<?',
     onFinish: '&',
     onCancel: '&',
-    wizardReady: '=?',
-    wizardDone: '=?',
-    loadingWizardTitle: '=?',
-    loadingSecondaryInformation: '=?',
-    embedInPage: '=?',
+    wizardReady: '<?',
+    wizardDone: '<?',
```

Comment:
```wizardReady``` and ```wizardDone``` don't need their statuses piped back out?

Reply:
no, these are just to inform the wizard of the app's status.